tonic_validate/metrics/duplication_metric.py

```diff
@@ -11,11 +11,13 @@
 
 
 class DuplicationMetric(BinaryMetric):
-    """Checks whether or not there's duplicate information in the response."""
-
     name: str = "duplication_metric"
 
     def __init__(self):
+        """
+        Binary metric that checks whether or not there's duplicate information in the response.
+        Returns either 1 (contains duplicate information) or 0 (does not contain duplicate information).
```

Comment:
Returns 1 (True) if the response contains duplicate information. Returns 0 (False) if it does not contain duplicate information.